cmd/lotus/daemon.go

```diff
@@ -98,9 +98,24 @@ var DaemonCmd = &cli.Command{
 			Name:  "genesis",
 			Usage: "genesis file to use for first node run",
 		},
+		// FIXME: Consider removing this flag in future releases (deprecated
+		//  in favor of "isolated" below). `bootstrap` is ambiguous: first it
+		//  is already the (only) possible value of `--profile` which signals
+		//  if this is a bootstrap node, and second, `bootstrap` here means
+		//  "connect to bootstrap nodes" which even if technically correct
+		//  the more descriptive conclusion is that, if false, the node won't
+		//  connect to *any* nodes at all (since you need the bootstrap nodes
+		//  to find the others).
 		&cli.BoolFlag{
-			Name:  "bootstrap",
-			Value: true,
+			Name:   "bootstrap",
+			Value:  true,
+			Usage:  "DEPRECATED: use 'isolated' flag instead",
+			Hidden: true,
+		},
+		&cli.BoolFlag{
+			Name:  "isolated",
```

Comment:
`--bootstrap=false` doesn't guarantee that the node won't connect to other nodes without explicitly asking it to, which `isolated`implies - if some other node connects to us we'll likely automatically connect to other nodes we get from pubsub/dht

Reply:
Makes sense, what other name would be more appropriate? (Alternatively maybe we want to make it more isolated and should turn off other services besides the peer manager.)